packages/ffe-form-react/src/PhoneNumber.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+import uuid from 'uuid';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumber({
+                                        number,
+                                        countryCode,
+                                        onCountryCodeChange,
+                                        onNumberChange,
+                                        onCountryCodeBlur,
+                                        onNumberBlur,
+                                        locale,
+                                        disabled,
+                                        countryCodeInvalid,
+                                        numberInvalid,
+                                        className
+                                    }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    const numberId = uuid.v4();
+    const countryCodeId = uuid.v4();
+
+    return (
+        <div className={`ffe-input-group ${className ? className : ''}`}>
+            <div className="ffe-phone-number">
+                <div className="ffe-phone-number__country-code">
+                    <label className="ffe-form-label" htmlFor={countryCodeId}>{text.COUNTRY_CODE}</label>
+                    <div className="ffe-phone-number__input-group">
+                        <span className="ffe-phone-number__plus">+</span>
+                        <input
+                            id={countryCodeId}
+                            className="ffe-input-field ffe-phone-number__country-code-input"
+                            type="tel"
+                            disabled={disabled}
+                            value={countryCode}
+                            aria-invalid={countryCodeInvalid}
+                            onChange={onCountryCodeChange}
+                            onBlur={onCountryCodeBlur}
+                        />
+                    </div>
+                </div>
+                <div className="ffe-phone-number__number">
+                    <label className="ffe-form-label" htmlFor={numberId}>{text.PHONE_NUMBER}</label>
+                    <input
+                        id={numberId}
+                        type="tel"
+                        className="ffe-input-field ffe-phone-number__phone-input"
+                        onChange={onNumberChange}
+                        onBlur={onNumberBlur}
+                        value={number}
+                        aria-invalid={numberInvalid}
+                        disabled={disabled}
+                    />
```

Comment:
In order to stop adding props at some point - perhaps we can pass in props to these two input fields in their own objects? It has the downside of having a prop being an object, but it lets us spread "unknown" props. Any "unknown props outside of these two would be spread on both, making sure you could add i.e. `aria-invalid` or `disabled` props without having to repeat yourself.

In other words, an API that would look like this:

```js
<PhoneNumberInput 
  countryCodeProps={{ value: '47', className: 'extra-country-code-class' }}
  phoneNumberProps={{ value: '22225555', onChange: props.onChange }}
  aria-invalid="false" // for both input fields
/> 
```

Reply:
It would be a breaking change to refactor this out later though - why not make the API as flexible as possible while we're designing it? That being said, if every one else is okay with this, I'm not going to argue on about it :)